src/content/docs/installation/method/uninstall-macos.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: macOS Uninstallation
+---
+import Aside from "@components/Aside.astro";
+export const partial = true;
+
+You can uninstall Daytona after using the official installation script.
+
+<Aside type="caution">
+This procedure is destructive and irreversible.
+Ensure you have a back up of any Workspace data and relevant configuration before proceeding.
+</Aside>
+
+__Prerequisites__
+
+* A POSIX-compatible shell such as bash or zsh.
+
+__Procedure__
+
+1. Run the following commands to purge local Daytona data and remove the binary:
+
+    ```shell
+    daytona purge
+    sudo rm $(where daytona)
+    ```
+
+    Run the following command if the `daytona purge` command does not work, or if you want to manually purge local Daytona data and remove the binary:
```

Comment:
This command wont't delete the binary. `sudo rm $(where daytona)` is still needed